images/blsq/Dockerfile

```diff
@@ -21,10 +21,9 @@ RUN conda install --yes -c R \
   conda clean --all -y
 
 
-USER ${NB_UID}
```

Comment:
A package (rgdal ?) needs to update the gdal version installed and fails due to a permission denied.